meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-firmware/imx-firmware-lf-5.10.bb

```
@@ -0,0 +1,24 @@
+# Copyright 2021 (C) Foundries.IO LTD
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://${S}/EULA.txt;md5=e565271ec9a80ce47abbddc4bffe56fa"
+
+S = "${WORKDIR}/git"
+
+SRC_URI = " \
+    git://github.com/NXP/imx-firmware.git;protocol=https;branch=lf-5.10.52_2.1.0 \
+"
+SRCREV = "6d7f77b83164b08334806c4aa2034bc1f7da7b7d"
+
+PACKAGES = "${PN}"
+RPROVIDES_${PN} = "imx-firmware"
+
+FILES_${PN} = " \
+	${nonarch_base_libdir}/firmware/mrvl/* \
```

I believe we use 4 spaces instead of a tab, could you please check?
```suggestion
    ${nonarch_base_libdir}/firmware/mrvl/* \
```

---

Thank you, fixed!